package.json

```diff
@@ -42,6 +42,8 @@
     "lz-string": "^1.4.4",
     "mobx": "5.15.7",
     "mobx-react": "6.3.0",
+    "perftrace": "^1.3.0",
+    "prettier": "^3.0.0",
```

Comment:
```suggestion
```

I think this is a merge conflict artifact from https://github.com/tinyspeck/sleuth/pull/246